### PR TITLE
Updated to 1.21.3

### PR DESCRIPTION
**Pull request by thepapanoob**
Hey Tom :)

i updated the mod to 1.21.3.
All i had to do is update the versions in gradle.properties & update gradle wich i just copied from the fabric mod template.
Everything else was still up to date 👍

Sincerely Papanoob

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -5,20 +5,20 @@ org.gradle.parallel=true
...
-loader_version=0.15.11
+minecraft_version=1.21.3
+yarn_mappings=1.21.3+build.2
+loader_version=0.16.9
 
 # Mod Properties
 mod_id=bluemapsignmarkers
 mod_name=BlueMap Sign Markers
-mod_version=1.21-0.4.0
+mod_version=1.21.3-0.5.0
```
Comment (tpwalke2) [inline gradle.properties:15]: Thank you for updating the minor version.

File: gradle.properties [hunk 2]
```diff
@@ -5,20 +5,20 @@ org.gradle.parallel=true
...
 maven_group=com.tpwalke2.bluemapsignmarkers
 archives_base_name=bluemapsignmarkers
 
 # Dependencies
-fabric_version=0.100.4+1.21
-fabric_loom_version=1.7-SNAPSHOT
-blue_map_version=5.3
+fabric_version=0.107.3+1.21.3
+fabric_loom_version=1.8-SNAPSHOT
+blue_map_version=5.4
```
Comment (tpwalke2) [inline gradle.properties:23]: Do you know if BlueMap have updated to 1.21.3?

**Review by tpwalke2 — COMMENTED** [review]
Thank you for the PR.

Overall it looks good. I just have a question about the BlueMap update.

File: gradle.properties (hunk 2 above)
Comment (thepapanoob) [reply gradle.properties:23]: they did! im using it on my server :)

**Review by tpwalke2 — APPROVED** [review]

**Pull request merged**